src/store/reducer/__tests__/board.test.ts

```diff
@@ -0,0 +1,49 @@
+import {BoardState} from "types/store";
+import {AddColumnRequest, EditColumnRequest} from "types/column";
+import {boardReducer} from "../board";
+import {ActionFactory} from "../../action";
```

Comment:
pls no relative paths

Reply:
Fixed. (changed the imports also in `reducer/__tests__/users.test.ts`)